source/containers.tex

```diff
@@ -2625,9 +2625,9 @@
 \tcode{a_eq} denotes a value of type \tcode{X}
 when \tcode{X} supports multiple keys,
 \item
-\tcode{a_tran} denotes a possibly \keyword{const} value of type \tcode{X}
+\tcode{a_tran} denotes a value of type \tcode{X} or \tcode{const X}
 when the \grammarterm{qualified-id}
-\tcode{X::key_compare::is_transpa\-rent} is valid
+\tcode{X::key_compare::is_transparent} is valid
```

Comment:
This fits on a line now, so we don't need the hyphenation.

Reply:
Beautiful.